packages/rocketchat-i18n/i18n/es.i18n.json

```diff
@@ -2122,7 +2160,6 @@
   "preview-c-room": "Vista previa del canal público",
   "preview-c-room_description": "Permiso para ver los contenidos de un canal público antes de unirse",
   "Privacy": "Privacidad",
-  "Privacy_Policy": "Política de Privacidad",
```

Comment:
Even though this message is "duplicated" (exists with another key), it shouldn't be removed, as it is still used in the app.

Reply:
You are right. I will change it!